resources/js/functions.js

```diff
@@ -234,33 +234,52 @@ $(document).ready(function() {
 
 // Scala in the browser
 $(document).ready(function() {
-    if ($("#scastie-textarea").length) {
-        var editor = CodeMirror.fromTextArea(document.getElementById("scastie-textarea"), {
-            lineNumbers: true,
-            matchBrackets: true,
-            theme: "monokai",
-            mode: "text/x-scala",
-            autoRefresh: true,
-            fixedGutter: false
-          });
-        editor.setSize("100%", ($("#scastie-code-container").height()));
-
-        var codeSnippet = "List(\"Hello\", \"World\").mkString(\"\", \", \", \"!\")";
-        editor.getDoc().setValue(codeSnippet);
-        editor.refresh();
-
-        $('.btn-run').click(function() {
-            // TODO: Code to connect to the scastie server would be here, what follows is just a simulation for the UI elements:
-            $('.btn-run').addClass("inactive");
-            $('.btn-run i').removeClass("fa fa-play").addClass("fa fa-spinner fa-spin");
-            setTimeout(function() {
-              var currentCodeSnippet = editor.getDoc().getValue();
-              console.log("Current code snippet: " + currentCodeSnippet);
-              $('.btn-run').removeClass("inactive");
-              $('.btn-run i').removeClass("fa-spinner fa-spin").addClass("fa fa-play");
-            }, 2000);
-        })
+  if ($("#scastie-textarea").length) {
+    var editor = 
+      CodeMirror.fromTextArea(
+        document.getElementById("scastie-textarea"),
+        {
+          // lineNumbers: false,
+          matchBrackets: true,
+          theme: "monokai",
+          mode: "text/x-scala",
+          autoRefresh: true,
+          fixedGutter: false,
+          extraKeys: {
+            'Ctrl-Enter': 'run',
+            'Cmd-Enter': 'run'
+          }
+        }
+      );
+
+    editor.setSize("100%", ($("#scastie-code-container").height()));
+
+    var codeSnippet = "List(\"Hello\", \"World\").mkString(\"\", \", \", \"!\")";
+    editor.getDoc().setValue(codeSnippet);
+    editor.refresh();
+
+    function run(){
+      console.log("run");
+      // var scastieBaseUrl = "https://scastie.scala-lang.org";
```

Comment:
I'm redeploying scastie, I will force push with the url pointing on the prod server.